pkg/auth/providers/oidc/oidc_client.go

```diff
@@ -18,23 +18,36 @@ func AddCertKeyToContext(ctx context.Context, certificate, key string) (context.
 		}
 		return oidc.ClientContext(ctx, certKeyClient), nil
 	}
+
```

Comment:
We're working towards adding more unit test over the codebase, it would be great if you could add some unit tests around this.